examples/controlplane/controlplane.yaml

```diff
@@ -25,6 +25,7 @@ spec:
   instanceType: ${CONTROL_PLANE_MACHINE_TYPE}
   iamInstanceProfile: "control-plane.cluster-api-provider-aws.sigs.k8s.io"
   sshKeyName: "${SSH_KEY_NAME}"
+  availabilityZone: ${AWS_REGION}a
```

Comment:
Not every region has a `a`, `b`, and `c` AZ (nor do they necessarily align between different accounts). Hardcoding these here may lead to non-functioning manifests for some users.

Reply:
> Not every region has a `a`, `b`, and `c` AZ (nor do they necessarily align between different accounts). Hardcoding these here may lead to non-functioning manifests for some users.

right, I will improve this script if this region has the a, b, c AZ , the script will generate the cluster yaml as before, if not, the script will set three masters with one az. what did you think?

Reply:
the az info as follows

eu-north-1: eu-north-1a eu-north-1b eu-north-1c
ap-south-1: ap-south-1a ap-south-1b ap-south-1c
eu-west-3: eu-west-3a eu-west-3b eu-west-3c
eu-west-2: eu-west-2a eu-west-2b eu-west-2c
eu-west-1: eu-west-1a eu-west-1b eu-west-1c
ap-northeast-2: ap-northeast-2a ap-northeast-2b ap-northeast-2c
ap-northeast-1: ap-northeast-1a ap-northeast-1c ap-northeast-1d
sa-east-1: sa-east-1a sa-east-1b sa-east-1c
ca-central-1: ca-central-1a ca-central-1b
ap-east-1: ap-east-1a ap-east-1b ap-east-1c
ap-southeast-1: ap-southeast-1a ap-southeast-1b ap-southeast-1c
ap-southeast-2: ap-southeast-2a ap-southeast-2b ap-southeast-2c
eu-central-1: eu-central-1a eu-central-1b eu-central-1c
us-east-1: us-east-1a us-east-1b us-east-1c us-east-1d us-east-1e us-east-1f
us-east-2: us-east-2a us-east-2b us-east-2c
us-west-1: us-west-1b us-west-1c
us-west-2: us-west-2a us-west-2b us-west-2c us-west-2d